tests/tests_pytorch/trainer/connectors/test_accelerator_connector.py

```diff
@@ -837,15 +839,15 @@ def get_defaults(cls):
         assert connector_default == trainer_defaults[name]
 
 
-@RunIf(min_cuda_gpus=1)  # trigger this test on our GPU pipeline, because we don't install the package on the CPU suite
-@pytest.mark.skipif(not package_available("lightning_colossalai"), reason="Requires Colossal AI Strategy")
-def test_colossalai_external_strategy(monkeypatch):
-    with mock.patch(
-        "lightning.pytorch.trainer.connectors.accelerator_connector._LIGHTNING_COLOSSALAI_AVAILABLE", False
-    ), pytest.raises(ModuleNotFoundError):
-        Trainer(strategy="colossalai")
-
-    from lightning_colossalai import ColossalAIStrategy
-
-    trainer = Trainer(strategy="colossalai", precision=16)
-    assert isinstance(trainer.strategy, ColossalAIStrategy)
+# @RunIf(min_cuda_gpus=1)  # trigger this test on our GPU pipeline, because we don't install the package on the CPU suite
```

Comment:
This is commented out here, since the validation for the precision argument is hardcoded in https://github.com/Lightning-AI/lightning-colossalai . I tried to first change it there, but these changes won't become effective until the strategy was released to pypi again.

After this was merged, I will post a follow-up PR in https://github.com/Lightning-AI/lightning-colossalai which will then be released and then have a follow-up PR to again add this test.

Discussed offline with @awaelchli